test/TestApps/AppLifecycleTestPackage/AppLifecycleTestPackage.wapproj

```diff
@@ -58,7 +58,13 @@
     <EntryPointProjectUniqueName>..\AppLifecycleTestApp\AppLifecycleTestApp.vcxproj</EntryPointProjectUniqueName>
     <GenerateAppxPackageOnBuild>true</GenerateAppxPackageOnBuild>
     <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
-    <PackageCertificateKeyFile>$(SolutionDir)temp\MSTest.pfx</PackageCertificateKeyFile>
+    <PackageCertificateKeyFile>$(SolutionDir).user\winappsdk.certificate.test.pfx</PackageCertificateKeyFile>
+<_SignMsix_UserDir>$(SolutionDir).user\</_SignMsix_UserDir>
```

Comment:
Are these still used here?

This project correctly uses the public Appx targets.  I don't think that utilizes _SignMsix* stuff.  The cert is the only thing that should be changed here, right?